app/sections/about/about.component.ts

```diff
@@ -1,39 +1,40 @@
-import {Component, OnInit} from 'angular2/core';
-import {RouteData, RouteParams} from 'angular2/router';
+import {Component, OnInit} from '@angular/core';
+import {ROUTER_DIRECTIVES, RouteData, RouteParams} from '@angular/router-deprecated';
 import {MarkdownPipe} from '../../shared/pipes/markdown.pipe';
-import {ROUTER_DIRECTIVES} from 'angular2/router';
 import {ContenfulContent} from '../../shared/services/contentful-content.service';
 import {NodePageContent} from '../../shared/structures/content-type.structures';
-import {Sidebar} from '../../shared/components/sidebar/sidebar.component.ts';
+import {SidebarComponent} from '../../shared/components/sidebar/sidebar.component.ts';
+import {ContentfulNodePage} from '../../shared/structures/aliases.structures';
 
-/**
- *
- */
 @Component({
-  template: <string> require('./about.component.html'),
-  directives: [...ROUTER_DIRECTIVES, Sidebar],
-  styles: [<string> require('./about.component.styl')],
+  template: require('./about.component.html') as string,
+  directives: [...ROUTER_DIRECTIVES, SidebarComponent],
+  styles: [require('./about.component.styl') as string],
   pipes: [MarkdownPipe]
 })
-export class About implements OnInit {
-  private content: NodePageContent;
+export class AboutComponent implements OnInit {
+  private content: ContentfulNodePage;
   private submenuItems: NodePageContent[] = [];
 
-  constructor(private _routerData: RouteData,
-              private _params: RouteParams,
-              private _contentfulContent: ContenfulContent) {
+  private _routerData: RouteData;
+  private _params: RouteParams;
+  private _contentfulContent: ContenfulContent;
+
+  public constructor(_routerData: RouteData,
```

Comment:
let's not use "_" symbol cause TS has access modifiers, hence there is no need to use this convention for private members